cookiecutter-xblock/{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/{{cookiecutter.package_name}}.py

```diff
@@ -24,7 +23,7 @@ class {{cookiecutter.class_name}}(XBlock):
 
     def resource_string(self, path):
         """Handy helper for getting resources from our kit."""
-        data = pkg_resources.resource_string(__name__, path)
+        data = resources.files(__package__).joinpath(path).read_bytes()
```

Comment:
We are calling the `xblock/utils/resources.py` methods where application and we can use them here. Please study below PR to follow it

https://github.com/openedx/xblock-in-video-quiz/pull/212/files#diff-cf5d4b07c1f2d28759f30b933fbdacaebf074f45fe08f1890b1fec6a9a049148R31

Reply:
Nice.